packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 - Zowe Explorer now includes support for the [VS Code display languages](https://code.visualstudio.com/docs/getstarted/locales) French, German, Japanese, Portuguese, and Spanish. [#3239](https://github.com/zowe/zowe-explorer-vscode/pull/3239)
 - Localization of strings within the webviews. [#2983](https://github.com/zowe/zowe-explorer-vscode/issues/2983)
+- Implemented an error correlation facility for providing user-friendly summaries of API and network errors. Extenders can contribute to the correlator to provide human-readable translations of error messages, as well as tips and additional resources for how to resolve the error. [#3243](https://github.com/zowe/zowe-explorer-vscode/pull/3243)
```

Comment:
Can we make it more action based?: "Apply (?) the new correlation facility to provide user-friendly summaries of API and network errors. Extenders can ... ."